apps/coscrad-frontend/src/components/footer/footer.tsx

```diff
@@ -22,6 +25,27 @@ export const Footer = (): JSX.Element => {
                 backgroundColor: '#ededed',
             }}
         >
+            <Grid>Phone: {phoneNumber}</Grid>
+            <Grid>Address: {address}</Grid>
+            <Grid item>
+                {internalLinks.map(({ url, iconUrl, description }) => (
+                    // <Tooltip title={description}>
+                    <IconButton
+                        href={url}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        color="primary"
+                    >
+                        {/* <img width={100} src={iconUrl} alt={description} /> */}
+                        {url.includes('facebook') && <FacebookOutlined />}
```

Comment:
Let's not use pattern-matching like this. It's best to be intentional. You need to update your internalLinks type to accommodate a 'platform' property. I might consider having a 
```ts
type SocialMediaLinks ={
 facebook?: string,
twitter?: string,
github?: string,
youtube?: string
}
```
Note that you still need to support links with dynamic logos. It is an organization requirement that there be a link to 
- tsilhqotin.ca
- tsilhqotinlanguage.ca

Reply:
Note that if you have a `socialMediaLinks` property on your content config with the above type, you can loop over the keys of this to create the actual icons.
```ts
[
[<FacebookOutlined />,'facebook'],
[<YouTube />, 'youtube'],
// ...
]
.map(
 ([Icon,platform]) =>[Icon,socialMediaLinks[platform]])
)
.filter(
([_,link]) => !isNullOrUndefinedLink
)
}
```
You might have to squeeze a unique key in while you're at it.